Command/UpdateCertificatesCommand.php

```diff
@@ -103,7 +103,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 'form_params' => [
                     'cert' => file_get_contents(sprintf('%s/fullchain.pem', $basePath)),
                     'key'  => file_get_contents(sprintf('%s/privkey.pem', $basePath)),
-                    'snis' => $domain,
+                    'snis[]' => $domain,
```

Comment:
Hmm looks like finally they changed the `snis` data format to an array.

Shouldn't this be `'snis' => [$domain]`?

Reply:
Nope, I tried that - Guzzle then sends it like this:
`&snis[0]=mydomain.com` - which Kong doesn't like

Reply:
Thanks for the lightning-quick review btw :) nice work on this!